### PR TITLE
Compacting C# 7 error codes

### PR DESCRIPTION
**Pull request by jcouv**
I was not able to fix all the instances of `CSnnnn` for the error codes that I updated, because some where already out-of-sync. So the error codes that appear in the comments in `VerifyDiagnostics` calls may not be correct.

To see all the error codes added for C# 7, run `git diff 11cef7ed5788acb67c74c53a35964a91ecac370a src/Compilers/CSharp/Portable/Errors/ErrorCode.cs` (that is the SHA for [VS 2015 update 3 micro-update 1](https://github.com/dotnet/roslyn/releases/tag/Visual-Studio-2015-Update-3-Micro-Update-1)).

Relates to https://github.com/dotnet/roslyn/issues/11405

### CODE REVIEW

File: src/Compilers/CSharp/Portable/Errors/ErrorCode.cs [hunk 1]
```diff
@@ -1293,8 +1293,6 @@ internal enum ErrorCode
         ERR_BlockBodyAndExpressionBody = 8057,
         ERR_FeatureIsExperimental = 8058,
         ERR_FeatureNotAvailableInVersion6 = 8059,
-        ERR_FeatureIsUnimplemented = 8060,
```
Comment (jcouv) [inline src/Compilers/CSharp/Portable/Errors/ErrorCode.cs]: This one was removed entirely, as it was unused.

**Comment by jcouv** [comment]
@gafter Could you do the first review?

**Comment by gafter** [comment]
Looking now.

**Comment by gafter** [comment]
:shipit:

**Comment by jcouv** [comment]
Thanks!
@VSadov This is another small review, if you don't mind.

**Comment by VSadov** [comment]
LGTM

**Comment by jcouv** [comment]
Thank you :-)
Going ahead with merge.

**Pull request merged**